source/fundamentals/authentication.txt

```diff
@@ -17,399 +17,65 @@ Authentication Mechanisms
-         {
-            ClientCertificates = new List<X509Certificate>()
-            {
-                new X509Certificate2("<path to X.509 certificate>", "<X.509 certificate password>")
-            }
-         };
-
-   .. tab:: MongoCredential
-      :tabid: default-mongo-credential
-
-      .. code-block:: csharp
-
-         var credential = MongoCredential.CreateMongoX509Credential("<X.509 certificate username>")
-         var settings = new MongoClientSettings 
-         {
-             Credential =  credential
-             SslSettings = new SslSettings
-             {
-                 ClientCertificates = new List<X509Certificate>()
-                 {
-                     new X509Certificate2("<path to X.509 certificate>", "<X.509 certificate password>")
-                 },
-             },
-             UseTls = true,
-             Server = new MongoServerAddress("<hostname", "<port>"),
-         };
-
-      .. note:: Certificate Type
-
-         Your certificate must be a :wikipedia:`PKCS #12<PKCS_12>` type certificate
-         with a ``.p12`` extension.
-
-      .. tip:: Username parameter
-
-         The username parameter provided to ``CreateMongoX509Credential`` must
-         match the distinguished subject name of your ``X.509`` certificate exactly.
-         You can alternatively pass ``null`` as the parameter to prompt the MongoDB
-         server to infer the username based on your ``X.509`` certificate.
-
-.. note::
-
-   If you are using Windows, the driver might be unable to locate an ``X.509``
-   authentication certificate created in memory. To learn more about a potential solution to
-   this issue, see the :ref:`csharp-troubleshooting-connection-X509`
-   section of the Connection Troubleshooting guide.
-
-API Documentation
------------------
-
-To learn more about any of the methods or types discussed in this
-guide, see the following API Documentation:
-
-- `MongoCredential() <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.MongoCredential.html>`__
-- `MongoClient() <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.MongoClient.html>`__
-- `MongoClientSettings <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.MongoClientSettings.html>`__
-- `CreateCredential() <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.MongoCredential.CreateCredential.html>`__
-- `CreateMongoX509Credential() <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.MongoCredential.CreateMongoX509Credential.html>`__
\ No newline at end of file
+   To learn how to establish a connection to your MongoDB deployment, see the
+   :ref:`csharp-connect-to-mongodb`.
+
+MongoDB Edition Compatibility
+-----------------------------
+
+The following table lists the authentication mechanisms supported by MongoDB and
+the {+mdb-server+} editions that each mechanism is compatible with. Click the name of
+a mechanism to learn more about how to use it with your application.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Authentication Mechanism
```

Comment:
Question for tech reviewer: Is this compat table accurate?

Reply:
Honestly I do not know, it's probably better to ask someone from server-side. Or @JamesKovacs might help with that.